book/go/api/word-count.md

```diff
@@ -215,4 +215,4 @@ The complete example is available [here](https://github.com/WallarooLabs/wallaro
 
 ### Next Steps
 
-For further reading, please refer to the [Wallaroo Go API Classes](api.md).
+Learn more about what as an application programmer need to do for [serialization](interworker-serialization-and-resilience.md) when writing a Wallaroo Go application. 
```

Comment:
> Learn more about what as an application programmer need to do for

This doesn't make sense.

Reply:
I missed a 'you'

Would that be good

"Learn more about what, as an application programmer, you need to do for"

Reply:
update pushed

Reply:
That would work, yes.